.github/workflows/ci.yml

```diff
@@ -0,0 +1,21 @@
+name: Rust
+
+on: [push, pull_request]
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest, macos-latest]
+
+    runs-on: ${{ matrix.os }}
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Build
+      run: cargo build --verbose
```

Comment:
If this is being built for consumers (instead of just for testing), you want

```cargo build --release```

At minimum, you should see a binary size difference (I'm seeing 3.3MB vs 987KB locally) and performance difference (although not overly noticeable for this use case, in my opinion)

Reply:
This is being built for tests. There will be another workflow for releasing.